toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -176,12 +176,12 @@ libcap-ng-0.7.9-4.cm2.x86_64.rpm
 libdb-5.3.28-5.cm2.x86_64.rpm
 libdb-devel-5.3.28-5.cm2.x86_64.rpm
 libdb-docs-5.3.28-5.cm2.x86_64.rpm
-rpm-4.14.2.1-4.cm2.x86_64.rpm
```

Comment:
Updating these versions will require you to also compile the new rpm version in the toolchain. Otherwise toolchain builds will break when these files are not found